### DISCUSSION TITLE
Regarding running WWIII using qsub command

### DISCUSSION THREAD

**Discussion by krishnanathira** [NOAA-EMC/WW3#1043] [Q&A] [answered]
Hi

I tried running the model this way
my submit.sh file is as follows,

```
#! /bin/bash
#PBS -l nodes=1:ppn=16
#PBS -N WW3-Run
#PBS -q default
#PBS -j oe
#PBS -V
cd /home/athirak/WW3-6.07.1/regtests 
cat $PBS_NODEFILE > pbsnodes
./bin/run_test -N -n 16 -p mpirun -s Ifremer1 -w output_test ../model A_test
```

"output_test" is my output folder and A_test is my work directory. I have kept all the inputs in one folder called "input" inside the A_test folder.
When I am running the model with qsub submit.sh, its terminating soon.
But, instead of using qsub, when I tried running the command ./bin/run_test -N -n 16 -p mpirun -s Ifremer1 -w output_test ../model A_test
in terminal, the model is running.

Can you please help me to solve this issue?
Many Thanks
Athira

**Comment by JessicaMeixner-NOAA** [comment]
Each HPC system is different and it's not possible for us to debug things here and I'd encourage you to reach out to your HPC help desk.  This does not appear to be a WW3 issue.

**Reply by krishnanathira** [reply]
Thank You!

**Comment by ukmo-ccbunney** [comment, marked as answer]
> Hi
> 
> I tried running the model this way my submit.sh file is as follows,
> 
> ```
> #! /bin/bash
> #PBS -l nodes=1:ppn=16
> #PBS -N WW3-Run
> #PBS -q default
> #PBS -j oe
> #PBS -V
> cd /home/athirak/WW3-6.07.1/regtests 
> cat $PBS_NODEFILE > pbsnodes
> ./bin/run_test -N -n 16 -p mpirun -s Ifremer1 -w output_test ../model A_test
> ```
> 
> "output_test" is my output folder and A_test is my work directory. I have kept all the inputs in one folder called "input" inside the A_test folder. When I am running the model with qsub submit.sh, its terminating soon. But, instead of using qsub, when I tried running the command ./bin/run_test -N -n 16 -p mpirun -s Ifremer1 -w output_test ../model A_test in terminal, the model is running.
> 
> Can you please help me to solve this issue? Many Thanks Athira

Sounds like a resource issue? Maybe you need to request more memory for the test?  Our PBS has quite a low memory default. Try `#PBS -l mem=2GB` or something similar (I don't know how much memory your model uses).

**Reply by krishnanathira** [reply]
Thanks. I will try that.

**Reply by krishnanathira** [reply]
It was a resource issue. Thanks for the help.
